### PR TITLE
4176 cpuinfo

### PR DESCRIPTION
**Pull request by sankhesh**
Provide more CPU information as requested in http://www.na-mic.org/Bug/view.php?id=4176

### CODE REVIEW

File: Base/QTGUI/qSlicerApplication.cxx [hunk 1]
```diff
@@ -828,11 +828,13 @@ void qSlicerApplication::displayApplicationInformations() const
...
-  qDebug("%s: %s %.3f MHz, %d cores",
+  unsigned int numberOfLogicalCPU = systemInfo->GetNumberOfLogicalCPU();
+
+  qDebug("%s: %s %s, %d cores, %d logical processors",
          qPrintable(titles.at(4).leftJustified(titleWidth, '.')),
          systemInfo->GetVendorString() ? systemInfo->GetVendorString() : "unknown",
-         systemInfo->GetProcessorClockFrequency()/1000,
-         numberOfPhysicalCPU);
+         systemInfo->GetModelName() ? systemInfo->GetModelName() : "unknown",
+         numberOfPhysicalCPU, numberOfLogicalCPU);
```
Comment (jcfr) [inline Base/QTGUI/qSlicerApplication.cxx]: Thanks for the PR.

Look like the frequency is not displayed anymore.

Comment (sankhesh) [inline Base/QTGUI/qSlicerApplication.cxx]: Its usually included in the model name.
This is what I see on Linux and Mac:

```
CPU ......................: GenuineIntel Intel(R) Xeon(R) CPU E5-2630 0 @ 2.30GHz, 12 cores, 24 logical processors
```

Comment (jcfr) [inline Base/QTGUI/qSlicerApplication.cxx]: Got it ... it was mention in the commit mesg. 

On your machine, does this correctly describe the before and after ?  Could you then amend the last commit including it ?

```
Before:
CPU ......................: GenuineIntel 2.300 MHz, 1 cores

After:
CPU ......................: GenuineIntel Intel(R) Xeon(R) CPU E5-2630 0 @ 2.30GHz, 12 cores, 24 logical processors
```

**Comment by jcfr** [comment]
Thanks for the PR :+1: 

Integrated as [r25141](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=25141)  and [r25142](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=25142)

**Pull request closed without merging**